test/dotnet-new3.UnitTests/DotnetNewHelp.cs

```diff
@@ -102,6 +102,22 @@ public class DotnetNewHelp
                   bool - Optional                                                    
                   Default: true                                                      ";
 
+        private const string ConsoleAppHelp =
+@"Console Application (C#)
```

Comment:
this is example of `console-app -h` output. 
note that it doesn't have framework, as framework has only single choice `net6.0`.